frontend/ICard/README.md

```diff
@@ -0,0 +1,31 @@
+# ICard Mobile Application
+
+## Installation
+
+Type in terminal (assuming that the developers have already install Node.js)
+
+```bash
+npm install
+```
+
+## Running Application
+
+Type in terminal to start Metro
+
+```bash
+npx react-native start
+```
+
+### Android
```

Comment:
Add in README.md instructions to clean gradle when running apps in android https://stackoverflow.com/questions/42570067/react-native-always-gradlew-clean-before-run-react-native-run-android

Reply:
Added with https://github.com/UofA-Blueprint/ICard/pull/1/commits/612552850bce1a529d2b705c2b8db5e583465a68